doc/api/errors.md

```diff
@@ -563,6 +570,67 @@ found [here][online].
   encountered by [`http`][] or [`net`][] -- often a sign that a `socket.end()`
   was not properly called.
 
+<a id="nodejs-error-codes"></a>
+## Node.js Error Codes
+
+<a id="ERR_INVALID_ARG_TYPE"></a>
+### ERR_INVALID_ARG_TYPE
+
+The `'ERR_INVALID_ARG_TYPE'` error code is used generically to identify that
+an argument of the wrong type has been passed to a Node.js API.
+
+<a id="ERR_INVALID_CALLBACK"></a>
+### ERR_INVALID_CALLBACK
+
+The `'ERR_INVALID_CALLBACK'` error code is used generically to identify that
+a callback function is required and has not been provided to a Node.js API.
+
+<a id="ERR_REPL_INVALID_EVAL_CONFIG"></a>
+### ERR_REPL_INVALID_EVAL_CONFIG
+
+The `'ERR_REPL_INVALID_EVAL_CONFIG'` error code is used when both
+`breakEvalOnSigint` and `eval` options are set in the REPL config.
+Allowing both of these would be confusing as `breakEvalOnSigint`
+does not influence the same thing as the `eval` option does.
+
+<a id="ERR_REPL_OLD_HISTORY_PARSING_ERROR"></a>
+### ERR_REPL_OLD_HISTORY_PARSING_ERROR
+
+The `'ERR_REPL_OLD_HISTORY_PARSING_ERROR'` error code is used when parsing
```

Comment:
Probably `ERR_INVALID_OLD_REPL_HISTORY`? The `_ERROR` at the end is unnecessary.